demo/witness/worker/worker.js

```diff
@@ -247,13 +258,13 @@ async function stmt(request) {
       const res = await statement(REBASE_SK, body_str, JSON.stringify(opts));
       return new Response(res, {status: 200, headers: headers});
     } else {
-      return new Response(JSON.stringify(new Error(`Expected content-type application/json, got: ${contentType}`)), {
+      return new Response(JSON.stringify({ error: `Expected content-type application/json, got: ${contentType}`}), {
```

Comment:
should this response have it's content type set to application/json? or is that done elsewhere

Reply:
I think it's done elsewhere. I'm not sure, I'll take this down as a note because I'm going to be doing some error reporting improvements to this and the repos that touch it, and I'll look then. If it's only implied, it should explicitly set.